stdlib/public/Platform/Windows/SwiftWinDbg/SwiftWinDbg.def

```diff
@@ -0,0 +1,7 @@
+LIBRARY "SwiftWinDbg.dll"
+
+EXPORTS
+  WinDbgExtensionDllInit
+  ExtensionApiVersion
+  help
+  readstring
```

Comment:
Is there a reason to use the def file rather than just marking the entry points with `__declspec(dllexport)`?

Reply:
Sorry, I get exceptions when I try to do this for some reason.

Reply:
Exceptions?

Reply:
If I append `__declspec(dllexport)` to a function declaration and remove it from `SwiftWinDbg.def`, WinDbg claims that the function does not exist. So I don't think the functions are getting exported without the `.def` file.